modules/wmco-disconnected-cluster.adoc

```diff
@@ -4,25 +4,31 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="wmco-disconnected-cluster_{context}"]
-= Using Windows containers in a disconnected cluster
+= Using Windows containers with a mirror registry
 
-The Windows Machine Config Operator (WMCO) can be installed into a {product-title} cluster in a disconnected, or air-gapped, network. A _disconnected network_ is a restricted network without direct internet connectivity.
+The Windows Machine Config Operator (WMCO) can pull images from a registry mirror rather than from a public registry. You can pull images from the mirror registry directly or you can use you can use an `ImageDigestMirrorSet` (IDMS) or `ImageTagMirrorSet` (ITMS) object to configure your cluster to pull images from the mirror registry.
```

Comment:
`You can pull images from the mirror registry directly` is not accurate, WMCO only enables pulling from mirror registries if IDMS/ITMS are in use

Reply:
@saifshaikh48 @rrasouli I was referring to Ronnie's comment here: [we allow pulling an image directly from a disconnected registry](https://github.com/openshift/openshift-docs/pull/74918/files/6ed77c32066629dfd6520bd883a89117831aca7d#diff-5fa1176e0fd07350c91b903bf4d7eb0f9fa8eaa37343148a503fcb82823b6da6).  Users can pull directly from a disconnected registry that is not mirrored, but not from one that is mirrored?